### PR TITLE
Fix `themeChanged` closure strong reference cycles

### PR DESCRIPTION
**Pull request by Jim-0**
Learn more about, see https://learnappmaking.com/how-to-fix-strong-reference-cycle-swift/
Refer https://github.com/cntrump/HighlightrObjC/blob/844e3d44ed80e71da6cac5b240184d99528fddaa/Highlightr/Highlightr/CodeAttributedString.m

### CODE REVIEW

File: Pod/Classes/CodeAttributedString.swift [hunk 1]
```diff
@@ -208,10 +208,10 @@ open class CodeAttributedString : NSTextStorage
     
     func setupListeners()
     {
-        highlightr.themeChanged =
-            { _ in
-                    self.highlight(NSMakeRange(0, self.stringStorage.length))
-            }
+        weak var wself: CodeAttributedString? = self
+        highlightr.themeChanged = { _ in
+            wself?.highlight(NSMakeRange(0, wself?.stringStorage.length ?? 0))
```
Comment (raspu) [inline Pod/Classes/CodeAttributedString.swift:213]: Good catch, but I'll prefer this pattern instead:
```suggestion
        highlightr.themeChanged =
            { [weak self] _ in
                    guard let self = self else { return }
                    self.highlight(NSMakeRange(0, self.stringStorage.length))
```

Comment (Jim-0) [reply Pod/Classes/CodeAttributedString.swift:213]: Well, It's more safe using `guard else`

Comment (Jim-0) [reply Pod/Classes/CodeAttributedString.swift]: Well, It's safer using `guard else`

Comment (Jim-0) [reply Pod/Classes/CodeAttributedString.swift]: Well, It has better code extensibility.

**Pull request merged**